translator/internaldata/metrics_to_oc.go

```diff
@@ -100,21 +112,41 @@ func collectLabelKeys(metric data.Metric) *labelKeys {
 
 	// First, collect a set of all labels present in the metric
 	keySet := make(map[string]struct{})
+	md := metric.MetricDescriptor()
+	if !md.IsNil() {
+		addLabelKeys(keySet, md.LabelsMap())
```

Comment:
So this corresponds to the comment for this field that says it should apply to all points: https://github.com/open-telemetry/opentelemetry-proto/blob/e6c3c4a74d57f870a0d781bada02cb2b2c497d14/opentelemetry/proto/metrics/v1/metrics.proto#L177